### PR TITLE
Add Ionic (React) & Capacitor interactive quickstart

### PR DESCRIPTION
**Pull request by stevehobbsdev**
image

### CODE REVIEW

**Review by frederikprijck — DISMISSED** [review]

File: articles/quickstart/native/ionic-react/interactive.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+<%= include('../_includes/ionic/_article_intro') %>
+
+<%= include('../_includes/_getting_started', { library: 'Ionic' }) %>
+
+<%= include('../_includes/ionic/_configure_urls') %>
+
+<%= include('../../_includes/_auth0-react-install.md') %>
+
+<%= include('../_includes/ionic/_install_plugins') %>
```
Comment (frederikprijck) [inline articles/quickstart/native/ionic-react/interactive.md:35]: Would it make sense to use the `../../_includes/_configure_auth0_interactive.md` here? That one refers to the interactive application configurator that is part of the new design. Users can now setup apps and its settings through that.

Comment (stevehobbsdev) [reply articles/quickstart/native/ionic-react/interactive.md:35]: Makes sense, let me have a look.

Comment (stevehobbsdev) [reply articles/quickstart/native/ionic-react/interactive.md:35]: As mentioned on Slack, not sure I can use this include as I need to use `${account.namespace}` and other expansions but when I do that I get an "unexpected token" error.

Comment (stevehobbsdev) [reply articles/quickstart/native/ionic-react/interactive.md:35]: @frederikprijck I copied the content out of that include into `_callback_urls` partial and modified it slightly for Ionic. Will need to share this with Angular but that's on a separate PR.

Looks like I'm getting the signup widget now too 👍🏻 

image

**Review by frederikprijck — CHANGES_REQUESTED** [review]
As mentioned on Slack, we are updating existing files that are used in the old quickstart. Even though that old quickstart shouldn't be used on auth0.com/docs anymore. It is still used in manage.auth0.com, so we shouldn't touch these files in such a way that they refer to interactive selector or anything that's part of the redesign.

**Comment by stevehobbsdev** [comment]
Resolved in 2560fddb6

**Review by frederikprijck — DISMISSED** [review]
Looks good, curious why we have an empty `_add_login_spa_note.md` file tho?

**Comment by stevehobbsdev** [comment]
I don't recall but doesn't look to be referenced from anywhere - removed in adaa5f6e0

**Review by frederikprijck — DISMISSED** [review]

**Review by avanscoy — DISMISSED** [review]

File: articles/quickstart/native/_includes/ionic/_configure_urls_interactive.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ Throughout this article, `YOUR_PACKAGE_ID` is your application's package ID. Thi
 
 ### Configure an application
 
-Use the interactive selector to create a new Auth0 application or select an existing application that represents the project with which you want to integrate. Every application in Auth0 is assigned an alphanumeric, unique client ID that your application code will use to call Auth0 APIs through the SDK.
+Use the interactive selector to create a new Auth0 application or select an existing application that represents the project you want to integrate with. Every application in Auth0 is assigned an alphanumeric, unique client ID that your application code will use to call Auth0 APIs through the SDK.
 
-Any settings you configure using this quickstart will automatically update for your application in the <a href="${manage_url}/#/">Dashboard</a>, which is where you can manage your Applications in the future.
+Any settings you configure using this quickstart will automatically update for your Application in the <a href="${manage_url}/#/">Dashboard</a>, which is where you can manage your Applications in the future.
```
Comment (frederikprijck) [inline articles/quickstart/native/_includes/ionic/_configure_urls_interactive.md]: Is this change intentional? Seems like it should be `application` ?

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: articles/quickstart/native/_includes/ionic/_configure_urls_interactive.md (hunk 2 above)
Comment (stevehobbsdev) [reply articles/quickstart/native/_includes/ionic/_configure_urls_interactive.md]: You're right, looks like I missed this when resolving the merge conflicts. Will fix

Comment (stevehobbsdev) [reply articles/quickstart/native/_includes/ionic/_configure_urls_interactive.md]: Fixed in 15eb75659

**Review by frederikprijck — APPROVED** [review]
